### ISSUE TITLE
Add new pools

### ISSUE THREAD

**Issue by Tomcatt** [Stratehm/stratum-proxy#57]
Hello,

Is there a way that not only I can add pools by the .config file but save pools entered from the web page http://127.0.0.1:8080  ?

Thanks.

**Comment by Stratehm** [comment]
No, there is no solution at the moment to save the current proxy configuration.
